packages/sfpowerscripts-cli/resources/schemas/sfdx-project.schema.json

```diff
@@ -267,11 +258,6 @@
       "title": "Post Install Script",
```

Comment:
Definition of postInstallScript hasn't been removed

Reply:
Removed